src/core/server/graph/schema/schema.graphql

```diff
@@ -8637,6 +8660,11 @@ type Subscription {
   """
   commentEdited(storyID: ID!): CommentEditedPayload!
 
+  """
+  commentRejected returns when a Comment has been rejected anywhere in the Story.
+  """
+  commentStatusChanged(storyID: ID!): CommentStatusChangedPayload!
```

Comment:
We want to expose minimal information about moderation to the public. Because we only care about rejected comments, this should be `commentRejected` instead.